ethereum/spec/src/main/java/tech/pegasys/teku/spec/datastructures/blocks/blockbody/versions/electra/BlindedBeaconBlockBodyElectraImpl.java

```diff
@@ -88,7 +91,8 @@ class BlindedBeaconBlockBodyElectraImpl
         syncAggregate,
         executionPayloadHeader,
         blsToExecutionChanges,
-        blobKzgCommitments);
+        blobKzgCommitments,
+        executionRequests);
```

Comment:
is this missing its correspondent get method?

Reply:
Maybe, but I did the bare minimum on blinded block. This is likely going to be addressed on https://github.com/Consensys/teku/issues/8624